python_ta/reporters/templates/template.txt

```diff
@@ -1,12 +1,105 @@
 <html>
 <head>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
 </head>
 <body>
-<u1>
-    {% for message in messages %}
-        <p>{{ message.msg_id }} ({{ message.symbol }}) {{ message.obj }}<br/>&emsp;&emsp;{{message.msg }}</p>
-    {% endfor %}
+	
+		<!-- Constantly updated date:
+		<script type="text/javascript">
+			document.write("<p>" + Date() + "</p>")
+		</script>
+		-->
+		<div id="preheader"> 
+			<!-- TODO: Needs to be fixed. -->
+			<p style="display: inline; text-align:left">
+				Module analysed: {{ mod_name }}
+			</p>
+			<span style="display: inline; text-align:right"> 
+				{{ date_time }}
+			</span>
+		</div>
+	
+	<div id="header">
+		<br/>
+		<div class="title">
+			PyTA Error Report
+		</div>
+	</div>
 
-</u1>
+	<br/>
+
+	<div id="inside">
+		<p/>
```

Comment:
Same comment as above - don't use empty `p` tags for spacing either. Set the `padding-top` for class `inside` to achieve the desired effect.